src/data-loaders/RelationalDataLoader.js

```diff
@@ -98,8 +99,34 @@ const convertPostgresSQLResponseToColumnMetaData = (rows) => {
     };
   });
 };
+
+const injectVariables = (statement, req) => {
+  const { variableMap } = req;
+  if (variableMap) {
```

Comment:
This expression needs to be negated, now it will never inject the variableMap.
```suggestion
  if (!variableMap) {
```